src/1-Profile/ProfileImageUpload.tsx

```diff
@@ -0,0 +1,133 @@
+import { DOMAIN } from "@env";
+import axios from "axios";
+import { Buffer } from "buffer";
+import React, { useState } from "react";
+import { StyleSheet, Text, View } from "react-native";
+import { ImageLibraryOptions, ImagePickerResponse, launchImageLibrary } from "react-native-image-picker";
+import { PROFILE_IMAGE_MIME_TYPES, ProfileImageUploadParams, StackNavigationProps } from "../TypesAndInterfaces/custom-types";
+import { useAppDispatch, useAppSelector } from "../TypesAndInterfaces/hooks";
+import { RootState, updateProfileImage } from "../redux-store";
+import theme, { COLORS } from "../theme";
+import { Outline_Button, Raised_Button } from "../widgets";
+
+const ProfileImageUpload = ({callback}:ProfileImageUploadParams):JSX.Element => {
+    const dispatch = useAppDispatch();
+
+    const [profileImage, setProfileImage] = useState("");
+    const [profileImageType, setProfileImageType] = useState("");
+
+    const jwt = useAppSelector((state: RootState) => state.account.jwt);
+    const userID = useAppSelector((state: RootState) => state.account.userID);
+    const userProfile = useAppSelector((state: RootState) => state.account.userProfile);
+
+    const RequestAccountHeader = {
+        headers: {
+          "jwt": jwt,
+          "Content-Type": "",
+        }
+      }
+
+    const postProfileImage = async () => {
+        // Set content type of the request so the server can process the blob
+        RequestAccountHeader["headers"]["Content-Type"] = profileImageType;
+  
+        if (profileImage !== "") {
+          // use ByteBuffer to decode the base64 string
+          await axios.post(`${DOMAIN}/api/user/` + userID + `/image/` + userID + '-user-profile.' + profileImageType.substring(6), Buffer.from(profileImage, 'base64'), RequestAccountHeader).then(response => {
```

Comment:
Why did you choose `base64` over `uri` or `origionalPath`?
On the portal web version, I was able to pass the file object directly into axios, is this not possible?